srv/salt/ceph/sysctl/default.sls

```diff
@@ -0,0 +1,14 @@
+
+/etc/sysctl.d/deepsea-aio-max-nr.conf:
```

Comment:
This is not a good filename, since without numbers, it's not clear in which order the - possibly conflicting - sysctls will be applied.

Reply:
Created #681